packages/calcite-components/src/components/date-picker/date-picker.stories.ts

```diff
@@ -97,6 +116,19 @@ export const rangeValuesNotInSameMonthAndYear_TestOnly = (): string => html`
   </script>
 `;
 
+export const rangeOneCalendarValuesNotInSameMonthAndYear = (): string => html`
+  <div style="width: 400px">
+    <calcite-date-picker range calendars="1"></calcite-date-picker>
+  </div>
+  <script>
+    (async () => {
+      await customElements.whenDefined("calcite-date-picker");
```

Comment:
Side note: we should investigate to see which options are available for screenshot test setup.